assets/css/_ladder.scss

```diff
@@ -21,6 +21,21 @@
   text-transform: uppercase;
 }
 
-.m-ladder__label {
-  font-size: 0.75rem;
+.m-ladder__vehicle {
+  cursor: pointer;
+}
+
+.m-ladder__vehicle-triangle {
+  fill: $blue-grey;
+  stroke: none;
+
+  &.selected {
+    stroke: $dark;
+    stroke-width: 2;
```

Comment:
We should decide which styling belongs in css and which styling belongs inline when writing the svg. I like this being here. Maybe stroke widths and colors should be in css, and shapes+positioning should be in svg? Would that be good to start standardizing on?

Reply:
Yeah, I think that makes sense. I guess I was sort of conceptualizing it as "put everything in CSS except where you can't", but I think that's in essence the same thing as what you are saying without being as clear.